<!--
*Summary of Emperor's Information*
Command executed on January 02, 2014 at 07:48:01
Emperor Version: 0.9.2-dev, master@7228fca
QIIME Version: 1.8.0-dev, master@3f16b2b
HostName: Antonios-MacBook-Pro.local
Metadata: /Users/antoniog/Downloads/IBD_Swedish/study_1070_split_library_seqs_and_mapping/study_1070_mapping_file.txt
Coordinates: /Users/antoniog/Downloads/IBD_Swedish/beta/pcoa/weighted_unifrac_16S_otu_table_even.txt
Command: /Users/antoniog/svn_programs/emperor/scripts/make_emperor.py -m study_1070_split_library_seqs_and_mapping/study_1070_mapping_file.txt -i beta/pcoa/weighted_unifrac_16S_otu_table_even.txt -o beta/emperor/weighted_unifrac_16S/
-->
<!doctype html>
<html lang="en">

<head>
    <title>Emperor</title>
    <meta charset="utf-8">
    <link rel="shortcut icon" href="emperor_required_resources/img/favicon.ico" />
    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
    <link rel="stylesheet" type="text/css" href="emperor_required_resources/emperor/css/emperor.css">
    <link rel="stylesheet" type="text/css" href="emperor_required_resources/css/jquery-ui2.css">
    <link rel="stylesheet" type="text/css" href="emperor_required_resources/css/colorPicker.css">
    <link rel="stylesheet" type="text/css" href="emperor_required_resources/css/spectrum.css">
    <link rel="stylesheet" type="text/css" href="emperor_required_resources/css/d3.parcoords.css">
    <table id="logotable" style="vertical-align:middle;text-align:center;height:100%;width:100%;margin:0;padding:0;border:0;">
        <tr><td><img src="emperor_required_resources/img/emperor.png" alt="Emperor" id="logo"/></td></tr>
    </table>
    <script type="text/javascript" src="emperor_required_resources/js/d3.v3.min.js"></script>
    <script type="text/javascript" src="emperor_required_resources/js/d3.parcoords.js"></script>
    <script type="text/javascript" src="emperor_required_resources/js/jquery-1.7.1.min.js"></script>
    <script type="text/javascript" src="emperor_required_resources/js/jquery-ui-1.8.17.custom.min.js"></script>
    <script src="emperor_required_resources/js/jquery.colorPicker.js"></script>
    <script src="emperor_required_resources/js/spectrum.js"></script>

    <script src="emperor_required_resources/js/Three.js"></script>
    <script src="emperor_required_resources/js/js/Detector.js"></script>
    <script src="emperor_required_resources/js/js/RequestAnimationFrame.js"></script>
    <script src="emperor_required_resources/emperor/js/emperor.js"></script>
    <script type="text/javascript" src="emperor_required_resources/js/THREEx.screenshot.js"></script>
    <script type="text/javascript" src="emperor_required_resources/js/FileSaver.min.js"></script>
    
    <script type="text/javascript">
    
var g_mappingFileHeaders = ['SampleID','BarcodeSequence','LinkerPrimerSequence','TARGET_SUBFRAGMENT','ASSIGNED_FROM_GEO','LAB_PERSON','DISEASE_STAT2','TITLE','RUN_PREFIX','AGE','INVESTIGATION_TYPE','HOST_COMMON_NAME','DEPTH','HOST_TAXID','SUBMIT_TO_INSDC','COMMON_NAME','INCLUDES_TIMESERIES','LONGITUDE','STUDY_ABSTRACT','BODY_SITE','PROJECT_NAME','ELEVATION','EXPERIMENT_CENTER','COLLECTION_DATE','ALTITUDE','ENV_BIOME','SEX','PLATFORM','COUNTRY','STUDY_TITLE','PRINCIPAL_INVESTIGATOR_CONTACT','HOST_SUBJECT_ID','TAXON_ID','SAMPLE_CENTER','SAMP_SIZE','PRINCIPAL_INVESTIGATOR','STUDY_DESCRIPTION','AGE_UNIT','MIENS_COMPLIANT','STUDY_ID','EXPERIMENT_DESIGN_DESCRIPTION','Description_duplicate','BODY_HABITAT','SEQUENCING_METH','RUN_DATE','ENV_MATTER','TARGET_GENE','DISEASE_STAT','ENV_FEATURE','KEY_SEQ','BODY_PRODUCT','AGE_IN_YEARS','RUN_CENTER','STUDY_ALIAS','TWIN_SIBLING','YEAR_OF_BIRTH','LIBRARY_CONSTRUCTION_PROTOCOL','LATITUDE','PMID','PAIR_NUMBER','REGION','STUDY_CENTER','Description'];
var g_mappingFileData = { 'H.p14.f27.446096': ['H.p14.f27.446096','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT03','71.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','27','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 14','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p13.b25.446019': ['H.p13.b25.446019','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19416','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 13','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.f29.445999': ['ICD.p15.f29.445999','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT04','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','29','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p31.f41.446029': ['CCD.p31.f41.446029','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19408','46.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','41','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','46','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','31','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p16.dc31.446046': ['ICD.p16.dc31.446046','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT13','53.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','31','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 16','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','53','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','16','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p1.f1.446017': ['H.p1.f1.446017','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19415','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 1, fecal','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1961','FLX','59.33','20816835','1','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p44.f86.446048': ['H.p44.f86.446048','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19413','39.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','86','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','39','KTH Stockholm, Sweden','jansson_twins_ibd','y','1969','FLX','59.33','20816835','44','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p18.f36.445998': ['H.p18.f36.445998','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT05','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','36','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 18','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p18.i33.446045': ['ICD.p18.i33.446045','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT08','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','33','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 18','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p14.b27.446004': ['H.p14.b27.446004','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19416','72.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','27','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 14','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:feces','72','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p26.f94.446069': ['H.p26.f94.446069','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19405','54.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','94','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','26','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p1.f8.446044': ['H.p1.f8.446044','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19415','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 1, fecal','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1961','FLX','59.33','20816835','1','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p13.f25.446075': ['H.p13.f25.446075','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT03','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 13','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.dc32.446088': ['CCD.p17.dc32.446088','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal and colonic Crohns disease with ulcerative colitis','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT13','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','32','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal and colonic Crohns disease with ulcerative colitis, pair 17','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p18.dc36.446000': ['H.p18.dc36.446000','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT14','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','36','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 18','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p34.f75.446051': ['H.p34.f75.446051','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19411','60.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','75','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','60','KTH Stockholm, Sweden','jansson_twins_ibd','y','1948','FLX','59.33','20816835','34','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.i32.446060': ['CCD.p17.i32.446060','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal and colonic Crohns disease with ulcerative colitis','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT08','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','32','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal and colonic Crohns disease with ulcerative colitis, pair 17','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p14.f28.446040': ['CCD.p14.f28.446040','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT03','71.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','28','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 14','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p39.f105.446035': ['H.p39.f105.446035','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19408','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1977','FLX','59.33','20816835','39','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p20.f43.446010': ['UC.p20.f43.446010','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19402','70.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','43','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','70','KTH Stockholm, Sweden','jansson_twins_ibd','y','1938','FLX','59.33','20816835','20','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p41.f102.446005': ['UC.p41.f102.446005','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-PROC','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19410','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','41','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p11.f26.446031': ['H.p11.f26.446031','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT03','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 11','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p14.b28.446036': ['CCD.p14.b28.446036','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19416','72.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','28','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 14','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','72','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.f32.446063': ['CCD.p17.f32.446063','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal and colonic Crohns disease with ulcerative colitis','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT04','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','32','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal and colonic Crohns disease with ulcerative colitis, pair 17','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:feces','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p11.i19.446012': ['CCD.p11.i19.446012','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT06','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 11','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p18.i36.446058': ['H.p18.i36.446058','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT09','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','36','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 18','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.dc30.446070': ['ICD.p15.dc30.446070','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT13','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','30','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p6.f12.446024': ['H.p6.f12.446024','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19415','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 6, fecal','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1951','FLX','59.33','20816835','6','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p37.f52.446039': ['UC.p37.f52.446039','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-EXT','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19406','65.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','52','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-LS','ENVO:human-associated habitat','TCAG','UBERON:feces','65','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','37','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p16.dc35.445984': ['H.p16.dc35.445984','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT14','53.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','35','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 16','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','53','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','16','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p43.f51.445986': ['UC.p43.f51.445986','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19412','55.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','51','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-LS','ENVO:human-associated habitat','TCAG','UBERON:feces','55','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','43','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p33.f83.446086': ['ICD.p33.f83.446086','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19410','62.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','83','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','62','KTH Stockholm, Sweden','jansson_twins_ibd','y','1946','FLX','59.33','20816835','33','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p14.i27.445997': ['H.p14.i27.445997','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT07','71.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','27','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 14','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p40.f73.446021': ['UC.p40.f73.446021','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19409','62.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','73','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','62','KTH Stockholm, Sweden','jansson_twins_ibd','y','1946','FLX','59.33','20816835','40','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p27.f64.446073': ['ICD.p27.f64.446073','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease with ulcerative colitis','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19404','55.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','64','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease with ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-PROC','ENVO:human-associated habitat','TCAG','UBERON:feces','55','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','27','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p40.f81.446055': ['H.p40.f81.446055','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19409','62.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','81','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','62','KTH Stockholm, Sweden','jansson_twins_ibd','y','1946','FLX','59.33','20816835','40','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p28.f55.446028': ['H.p28.f55.446028','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19405','61.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','55','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-EXT','ENVO:human-associated habitat','TCAG','UBERON:feces','61','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','28','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p23.f100.446009': ['ICD.p23.f100.446009','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19403','61.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','100','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','61','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','23','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.i34.446068': ['CCD.p17.i34.446068','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT08','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','34','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 17','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p13.dc24.446026': ['CCD.p13.dc24.446026','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT11','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 13','UBERON:colon','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p9.f17.446027': ['CCD.p9.f17.446027','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT01','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 9, fecal','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','9','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.dc34.446047': ['CCD.p17.dc34.446047','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT14','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','34','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 17','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p17.f34.445993': ['CCD.p17.f34.445993','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT05','31.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','34','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 17','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','31','KTH Stockholm, Sweden','jansson_twins_ibd','y','1976','FLX','59.33','20816835','17','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p35.f72.446033': ['CCD.p35.f72.446033','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19412','51.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','72','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','51','KTH Stockholm, Sweden','jansson_twins_ibd','y','1957','FLX','59.33','20816835','35','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p44.f54.446080': ['UC.p44.f54.446080','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19413','39.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','54','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-EXT','ENVO:human-associated habitat','TCAG','UBERON:feces','39','KTH Stockholm, Sweden','jansson_twins_ibd','y','1969','FLX','59.33','20816835','44','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p19.f82.446057': ['UC.p19.f82.446057','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-PROC','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19401','71.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','82','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1937','FLX','59.33','20816835','19','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p16.f35.446041': ['H.p16.f35.446041','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT05','53.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','35','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 16','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','53','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','16','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p42.f76.446066': ['UC.p42.f76.446066','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-PROC','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19411','57.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','76','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','57','KTH Stockholm, Sweden','jansson_twins_ibd','y','1951','FLX','59.33','20816835','42','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p34.f67.446052': ['CCD.p34.f67.446052','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19411','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1948','FLX','59.33','20816835','34','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICCD.p30.f57.446056': ['ICCD.p30.f57.446056','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','Icolonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19407','50.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','57','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','Icolonic Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-EXT','ENVO:human-associated habitat','TCAG','UBERON:feces','50','KTH Stockholm, Sweden','jansson_twins_ibd','y','1958','FLX','59.33','20816835','30','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p18.dc33.446023': ['ICD.p18.dc33.446023','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT14','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','33','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 18','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p38.f38.446094': ['UC.p38.f38.446094','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-PROC','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19407','44.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','38','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis, pair 38','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','44','KTH Stockholm, Sweden','jansson_twins_ibd','y','1964','FLX','59.33','20816835','38','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p10.i18.445987': ['ICD.p10.i18.445987','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT06','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 10, ileum','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','10','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p37.f113.446084': ['H.p37.f113.446084','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19406','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','37','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p39.f104.446074': ['UC.p39.f104.446074','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-EXT','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19408','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1977','FLX','59.33','20816835','39','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'uiCD.p32.f91.446020': ['uiCD.p32.f91.446020','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19409','43.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','91','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','43','KTH Stockholm, Sweden','jansson_twins_ibd','y','1965','FLX','59.33','20816835','32','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.dc29.446091': ['ICD.p15.dc29.446091','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT13','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','29','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:colon','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p20.f40.445982': ['UC.p20.f40.445982','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19402','70.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','40','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','70','KTH Stockholm, Sweden','jansson_twins_ibd','y','1938','FLX','59.33','20816835','20','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p21.f88.446034': ['ICD.p21.f88.446034','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19401','69.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','88','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','69','KTH Stockholm, Sweden','jansson_twins_ibd','y','1939','FLX','59.33','20816835','21','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p13.a24.445996': ['CCD.p13.a24.445996','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT09','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 13','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p12.dc22.446083': ['H.p12.dc22.446083','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT11','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 12','UBERON:colon','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','12','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p29.f48.446015': ['H.p29.f48.446015','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19406','53.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','48','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-PROC','ENVO:human-associated habitat','TCAG','UBERON:feces','53','KTH Stockholm, Sweden','jansson_twins_ibd','y','1955','FLX','59.33','20816835','29','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.f30.446076': ['ICD.p15.f30.446076','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT04','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','30','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.i29.446090': ['ICD.p15.i29.446090','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT07','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','29','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p33.f93.445992': ['H.p33.f93.445992','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19410','62.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','93','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','62','KTH Stockholm, Sweden','jansson_twins_ibd','y','1946','FLX','59.33','20816835','33','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p13.f24.445989': ['CCD.p13.f24.445989','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT02','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 13','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p12.f22.446097': ['H.p12.f22.446097','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT02','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 12','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','12','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p21.f101.445994': ['H.p21.f101.445994','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19401','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1939','FLX','59.33','20816835','21','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p35.f78.446089': ['H.p35.f78.446089','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19412','51.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','78','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','51','KTH Stockholm, Sweden','jansson_twins_ibd','y','1957','FLX','59.33','20816835','35','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p18.f33.446006': ['ICD.p18.f33.446006','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT05','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','33','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 18','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','18','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p32.f98.446030': ['H.p32.f98.446030','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19409','43.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','98','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','43','KTH Stockholm, Sweden','jansson_twins_ibd','y','1965','FLX','59.33','20816835','32','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p30.f85.445985': ['H.p30.f85.445985','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19407','50.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','85','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','50','KTH Stockholm, Sweden','jansson_twins_ibd','y','1958','FLX','59.33','20816835','30','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p22.f39.446093': ['ICD.p22.f39.446093','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19402','68.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','39','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','68','KTH Stockholm, Sweden','jansson_twins_ibd','y','1940','FLX','59.33','20816835','22','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p11.f19.445990': ['CCD.p11.f19.445990','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT01','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 11','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p10.f21.445991': ['ICD.p10.f21.445991','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT02','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 10','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','10','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p11.dc19.446067': ['CCD.p11.dc19.446067','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT10','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 11','UBERON:colon','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p25.f58.446081': ['H.p25.f58.446081','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19404','67.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','58','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-PROC','ENVO:human-associated habitat','TCAG','UBERON:feces','67','KTH Stockholm, Sweden','jansson_twins_ibd','y','1941','FLX','59.33','20816835','25','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p12.dc23.446065': ['CCD.p12.dc23.446065','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT11','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 12','UBERON:colon','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','12','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p24.f77.446082': ['UC.p24.f77.446082','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19403','60.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','77','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','60','KTH Stockholm, Sweden','jansson_twins_ibd','y','1948','FLX','59.33','20816835','24','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p14.i28.446037': ['CCD.p14.i28.446037','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT07','71.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','28','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 14','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1936','FLX','59.33','20816835','14','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p29.f110.446003': ['ICD.p29.f110.446003','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19406','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1955','FLX','59.33','20816835','29','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p42.f79.446022': ['H.p42.f79.446022','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19411','57.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','79','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','57','KTH Stockholm, Sweden','jansson_twins_ibd','y','1951','FLX','59.33','20816835','42','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p46.f71.446071': ['UC.p46.f71.446071','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-LS','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19414','52.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','unknown','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','71','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis','ENVO:human-associated habitat','TCAG','UBERON:feces','52','KTH Stockholm, Sweden','jansson_twins_ibd','y','1956','FLX','59.33','20816835','46','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p24.f68.446008': ['H.p24.f68.446008','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19403','60.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','68','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-LS','ENVO:human-associated habitat','TCAG','UBERON:feces','60','KTH Stockholm, Sweden','jansson_twins_ibd','y','1948','FLX','59.33','20816835','24','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p46.f69.446078': ['H.p46.f69.446078','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19414','52.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','unknown','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','69','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-PROC','ENVO:human-associated habitat','TCAG','UBERON:feces','52','KTH Stockholm, Sweden','jansson_twins_ibd','y','1956','FLX','59.33','20816835','46','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p26.f44.446002': ['UC.p26.f44.446002','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-EXT','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19405','54.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','44','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','26','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p38.f63.445988': ['H.p38.f63.445988','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19407','44.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','63','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-LS','ENVO:human-associated habitat','TCAG','UBERON:feces','44','KTH Stockholm, Sweden','jansson_twins_ibd','y','1964','FLX','59.33','20816835','38','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p10.i21.446062': ['ICD.p10.i21.446062','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT06','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 10','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','10','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p10.f18.446085': ['ICD.p10.f18.446085','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT01','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 10, fecal','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','10','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p11.i26.446092': ['H.p11.i26.446092','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT06','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 11','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p43.f42.446087': ['H.p43.f42.446087','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19412','55.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','42','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ileal Crohns disease','ENVO:human-associated habitat','TCAG','UBERON:feces','55','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','43','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p27.f103.446077': ['H.p27.f103.446077','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19404','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','27','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p11.b26.446032': ['H.p11.b26.446032','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19416','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 11','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1986','FLX','59.33','20816835','11','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p13.a25.446050': ['H.p13.a25.446050','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT09','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 13','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1943','FLX','59.33','20816835','13','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p15.i30.446059': ['ICD.p15.i30.446059','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT07','54.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:ileal mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','30','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 15','UBERON:ileum','Pyrosequencing','5/11/07','ENVO:excreta','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:ileal mucosa','54','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','15','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p45.f66.446001': ['UC.p45.f66.446001','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-PROC','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19414','38.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','66','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-PROC','ENVO:human-associated habitat','TCAG','UBERON:feces','38','KTH Stockholm, Sweden','jansson_twins_ibd','y','1970','FLX','59.33','20816835','45','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p9.f20.446011': ['CCD.p9.f20.446011','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT01','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 9','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','9','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p10.dc18.445983': ['ICD.p10.dc18.445983','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT10','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:colonic mucosa','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 10, descending colon','UBERON:colon mucosa','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:colonic mucosa','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','10','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p28.f53.446072': ['ICD.p28.f53.446072','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19405','61.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','53','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','61','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','28','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'CCD.p12.f23.446016': ['CCD.p12.f23.446016','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','colonic Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT02','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','colonic Crohns disease, pair 12','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1953','FLX','59.33','20816835','12','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p31.f61.446007': ['H.p31.f61.446007','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19408','46.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','61','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-EXT','ENVO:human-associated habitat','TCAG','UBERON:feces','46','KTH Stockholm, Sweden','jansson_twins_ibd','y','1962','FLX','59.33','20816835','31','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p41.f87.446038': ['H.p41.f87.446038','CGAT','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19410','61.0','mimarks-survey','flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','87','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','61','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','41','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p19.f70.446079': ['H.p19.f70.446079','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19401','71.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','70','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','ulcerative colitis-LS','ENVO:human-associated habitat','TCAG','UBERON:feces','71','KTH Stockholm, Sweden','jansson_twins_ibd','y','1937','FLX','59.33','20816835','19','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p22.f116.446042': ['ICD.p22.f116.446042','CGTA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19402','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','male','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1940','FLX','59.33','20816835','22','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'UC.p25.f115.446095': ['UC.p25.f115.446095','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ulcerative colitis-EXT','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19404','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ulcerative colitis','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1941','FLX','59.33','20816835','25','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p45.f80.446018': ['H.p45.f80.446018','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19414','38.0','mimarks-survey','prairie dog flea','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','80','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','38','KTH Stockholm, Sweden','jansson_twins_ibd','y','1970','FLX','59.33','20816835','45','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p23.f112.445995': ['ICD.p23.f112.445995','CATG','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19403','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1947','FLX','59.33','20816835','23','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'ICD.p16.f31.446054': ['ICD.p16.f31.446054','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','ileal Crohns disease','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','EYG96GT04','53.0','mimarks-survey','human','0','496922','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2007','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','31','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','years','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','ileal Crohns disease, pair 16','UBERON:feces','Pyrosequencing','5/11/07','ENVO:feces','16S rRNA','healthy','ENVO:human-associated habitat','TCAG','UBERON:feces','53','KTH Stockholm, Sweden','jansson_twins_ibd','y','1954','FLX','59.33','20816835','16','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'],'H.p6.f11.446043': ['H.p6.f11.446043','CTGA','CRRCACGAGCTGACGAC','V6','y','Cathy Lozupone/Jesse Stombaugh','healthy','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes','FBJN19415','None','mimarks-survey','None','0','None','n','human gut metagenome','0','18.06','BACKGROUND & AIMS:The composition of the gastrointestinal microbiota is thought to have an important role in the etiology of inflammatory bowel diseases (IBDs) such as Crohns disease (CD) and ulcerative colitis (UC). Interindividual variation and an inability to detect less abundant bacteria have made it difficult to correlate specific bacteria with disease. METHODS:We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease. RESULTS:The profiles of the microbial community differed with disease phenotypes; relative amounts of bacterial populations correlated with IBD phenotypes. The microbial compositions of individuals with CD differed from those of healthy individuals, but were similar between healthy individuals and individuals with UC. Profiles from individuals with CD that predominantly involved the ileum differed from those with CD that predominantly involved the colon; several bacterial populations increased or decreased with disease type. Changes specific to patients with ileal CD included the disappearance of core bacteria, such as Faecalibacterium and Roseburia, and increased amounts of Enterobacteriaceae and Ruminococcus gnavus.CONCLUSIONS:Bacterial populations differ in abundance among individuals with different phenotypes of CD. Specific species of bacteria are associated with ileal CD; further studies should investigate their role in pathogenesis. Copyright ?? 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.','UBERON:feces','jansson_twins_ibd','20.9','Karolinska Institute','2008','0.0','ENVO:human-associated habitat','female','FLX','GAZ:Sweden','A pyrosequencing study in twins shows that gastrointestinal microbial profiles vary with inflammatory bowel disease phenotypes.','jrjansson@lbl.org','None','408170','Karolinska Institute','.25, gram','Janet Jansson','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','unknown','y','1070','We used 454 pyrotag sequencing to determine the compositions of microbial communities in feces samples collected from a cohort of 40 twin pairs who were concordant or discordant for CD or UC, and in mucosal samples from a subset of the cohort. The cohort primarily comprised patients who were in remission, but also some with active disease.','healthy, pair 6, fecal','UBERON:feces','Pyrosequencing','6/3/08','ENVO:feces','16S rRNA','None','ENVO:human-associated habitat','TCAG','UBERON:feces','None','KTH Stockholm, Sweden','jansson_twins_ibd','y','1951','FLX','59.33','20816835','6','None','Karolinska Institute','A_pyrosequencing_study_in_twins_shows_that_gastrointestinal_microbial_profiles_vary_with_inflammatory_bowel_disease_phenotypes'] };

var g_spherePositions = new Array();
g_spherePositions['CCD.p14.i28.446037'] = { 'name': 'CCD.p14.i28.446037', 'color': 0, 'x': -0.308385, 'y': -0.104192, 'z': 0.023815, 'P1': -0.308385, 'P2': -0.104192, 'P3': 0.023815, 'P4': -0.019844, 'P5': 0.057688, 'P6': -0.063241, 'P7': 0.026824, 'P8': 0.003413, 'P9': -0.062876, 'P10': 0.041363 };
g_spherePositions['H.p14.i27.445997'] = { 'name': 'H.p14.i27.445997', 'color': 0, 'x': -0.137885, 'y': 0.111153, 'z': 0.023489, 'P1': -0.137885, 'P2': 0.111153, 'P3': 0.023489, 'P4': 0.066434, 'P5': 0.049408, 'P6': 0.001410, 'P7': -0.013071, 'P8': -0.035671, 'P9': -0.061876, 'P10': -0.005482 };
g_spherePositions['ICD.p15.i29.446090'] = { 'name': 'ICD.p15.i29.446090', 'color': 0, 'x': -0.014469, 'y': -0.046354, 'z': 0.068402, 'P1': -0.014469, 'P2': -0.046354, 'P3': 0.068402, 'P4': 0.036353, 'P5': 0.170924, 'P6': 0.018455, 'P7': -0.036116, 'P8': 0.033646, 'P9': 0.005716, 'P10': -0.069375 };
g_spherePositions['ICD.p15.i30.446059'] = { 'name': 'ICD.p15.i30.446059', 'color': 0, 'x': -0.150457, 'y': 0.097471, 'z': -0.026613, 'P1': -0.150457, 'P2': 0.097471, 'P3': -0.026613, 'P4': 0.072433, 'P5': 0.123249, 'P6': 0.083692, 'P7': -0.040890, 'P8': -0.034320, 'P9': 0.054932, 'P10': -0.012384 };
g_spherePositions['ICD.p16.dc31.446046'] = { 'name': 'ICD.p16.dc31.446046', 'color': 0, 'x': -0.210724, 'y': -0.021643, 'z': -0.019550, 'P1': -0.210724, 'P2': -0.021643, 'P3': -0.019550, 'P4': 0.079224, 'P5': 0.188381, 'P6': 0.059988, 'P7': 0.014810, 'P8': -0.031613, 'P9': 0.010302, 'P10': -0.042438 };
g_spherePositions['ICD.p15.dc30.446070'] = { 'name': 'ICD.p15.dc30.446070', 'color': 0, 'x': -0.138462, 'y': 0.108029, 'z': -0.035938, 'P1': -0.138462, 'P2': 0.108029, 'P3': -0.035938, 'P4': 0.041268, 'P5': 0.131660, 'P6': 0.087965, 'P7': -0.058440, 'P8': -0.035606, 'P9': 0.043140, 'P10': -0.005106 };
g_spherePositions['CCD.p17.dc32.446088'] = { 'name': 'CCD.p17.dc32.446088', 'color': 0, 'x': -0.407512, 'y': 0.078232, 'z': -0.103478, 'P1': -0.407512, 'P2': 0.078232, 'P3': -0.103478, 'P4': 0.076699, 'P5': -0.044135, 'P6': -0.042057, 'P7': -0.035224, 'P8': 0.055012, 'P9': 0.014964, 'P10': 0.033123 };
g_spherePositions['ICD.p15.dc29.446091'] = { 'name': 'ICD.p15.dc29.446091', 'color': 0, 'x': 0.171697, 'y': -0.058014, 'z': 0.047964, 'P1': 0.171697, 'P2': -0.058014, 'P3': 0.047964, 'P4': 0.068006, 'P5': 0.133029, 'P6': -0.057256, 'P7': -0.068523, 'P8': 0.037191, 'P9': 0.046463, 'P10': -0.052454 };
g_spherePositions['H.p26.f94.446069'] = { 'name': 'H.p26.f94.446069', 'color': 0, 'x': 0.299190, 'y': -0.285994, 'z': 0.073492, 'P1': 0.299190, 'P2': -0.285994, 'P3': 0.073492, 'P4': 0.016988, 'P5': 0.115529, 'P6': -0.080217, 'P7': 0.016846, 'P8': 0.161341, 'P9': 0.007522, 'P10': 0.004609 };
g_spherePositions['ICD.p28.f53.446072'] = { 'name': 'ICD.p28.f53.446072', 'color': 0, 'x': 0.150054, 'y': 0.149173, 'z': -0.036657, 'P1': 0.150054, 'P2': 0.149173, 'P3': -0.036657, 'P4': -0.248568, 'P5': 0.041683, 'P6': -0.093858, 'P7': -0.030838, 'P8': -0.038846, 'P9': -0.018981, 'P10': -0.072783 };
g_spherePositions['UC.p26.f44.446002'] = { 'name': 'UC.p26.f44.446002', 'color': 0, 'x': 0.001852, 'y': 0.075622, 'z': 0.046376, 'P1': 0.001852, 'P2': 0.075622, 'P3': 0.046376, 'P4': 0.033615, 'P5': 0.012484, 'P6': -0.003417, 'P7': -0.023796, 'P8': -0.038332, 'P9': -0.011205, 'P10': -0.004046 };
g_spherePositions['H.p28.f55.446028'] = { 'name': 'H.p28.f55.446028', 'color': 0, 'x': -0.096101, 'y': -0.165939, 'z': 0.144083, 'P1': -0.096101, 'P2': -0.165939, 'P3': 0.144083, 'P4': -0.106422, 'P5': 0.019053, 'P6': -0.020072, 'P7': 0.077810, 'P8': 0.025805, 'P9': 0.051879, 'P10': 0.051221 };
g_spherePositions['UC.p19.f82.446057'] = { 'name': 'UC.p19.f82.446057', 'color': 0, 'x': -0.069928, 'y': 0.065612, 'z': 0.085677, 'P1': -0.069928, 'P2': 0.065612, 'P3': 0.085677, 'P4': -0.056945, 'P5': -0.036979, 'P6': 0.025276, 'P7': -0.000354, 'P8': 0.003890, 'P9': -0.060607, 'P10': -0.034808 };
g_spherePositions['H.p21.f101.445994'] = { 'name': 'H.p21.f101.445994', 'color': 0, 'x': 0.037468, 'y': 0.076664, 'z': 0.071121, 'P1': 0.037468, 'P2': 0.076664, 'P3': 0.071121, 'P4': 0.031546, 'P5': 0.007508, 'P6': -0.033520, 'P7': -0.003811, 'P8': -0.038389, 'P9': -0.006255, 'P10': -0.031287 };
g_spherePositions['H.p19.f70.446079'] = { 'name': 'H.p19.f70.446079', 'color': 0, 'x': 0.050931, 'y': -0.033650, 'z': 0.129228, 'P1': 0.050931, 'P2': -0.033650, 'P3': 0.129228, 'P4': -0.066490, 'P5': -0.050838, 'P6': 0.060901, 'P7': -0.040482, 'P8': 0.021186, 'P9': 0.023637, 'P10': -0.032113 };
g_spherePositions['ICD.p21.f88.446034'] = { 'name': 'ICD.p21.f88.446034', 'color': 0, 'x': -0.260889, 'y': -0.650900, 'z': -0.214674, 'P1': -0.260889, 'P2': -0.650900, 'P3': -0.214674, 'P4': -0.034056, 'P5': -0.146971, 'P6': 0.086930, 'P7': -0.049020, 'P8': -0.077198, 'P9': -0.038684, 'P10': -0.151573 };
g_spherePositions['CCD.p13.a24.445996'] = { 'name': 'CCD.p13.a24.445996', 'color': 0, 'x': -0.367906, 'y': -0.003456, 'z': -0.059563, 'P1': -0.367906, 'P2': -0.003456, 'P3': -0.059563, 'P4': 0.051784, 'P5': 0.050643, 'P6': -0.044576, 'P7': 0.014344, 'P8': -0.001390, 'P9': -0.026333, 'P10': -0.008435 };
g_spherePositions['H.p13.a25.446050'] = { 'name': 'H.p13.a25.446050', 'color': 0, 'x': -0.275564, 'y': 0.085680, 'z': -0.039776, 'P1': -0.275564, 'P2': 0.085680, 'P3': -0.039776, 'P4': 0.085690, 'P5': 0.052007, 'P6': -0.007316, 'P7': 0.018779, 'P8': -0.008122, 'P9': -0.038607, 'P10': -0.015315 };
g_spherePositions['H.p18.i36.446058'] = { 'name': 'H.p18.i36.446058', 'color': 0, 'x': -0.283185, 'y': 0.050287, 'z': -0.017247, 'P1': -0.283185, 'P2': 0.050287, 'P3': -0.017247, 'P4': 0.021113, 'P5': -0.012587, 'P6': -0.007254, 'P7': -0.000474, 'P8': 0.020855, 'P9': -0.035874, 'P10': -0.009293 };
g_spherePositions['UC.p24.f77.446082'] = { 'name': 'UC.p24.f77.446082', 'color': 0, 'x': 0.275076, 'y': 0.029848, 'z': -0.028330, 'P1': 0.275076, 'P2': 0.029848, 'P3': -0.028330, 'P4': 0.111007, 'P5': 0.016019, 'P6': -0.046854, 'P7': 0.027689, 'P8': -0.037085, 'P9': 0.025258, 'P10': 0.023583 };
g_spherePositions['H.p24.f68.446008'] = { 'name': 'H.p24.f68.446008', 'color': 0, 'x': 0.414451, 'y': 0.083183, 'z': -0.133151, 'P1': 0.414451, 'P2': 0.083183, 'P3': -0.133151, 'P4': -0.098770, 'P5': -0.007901, 'P6': -0.035761, 'P7': 0.035312, 'P8': 0.006343, 'P9': -0.021730, 'P10': 0.055203 };
g_spherePositions['ICD.p23.f112.445995'] = { 'name': 'ICD.p23.f112.445995', 'color': 0, 'x': 0.323678, 'y': -0.091744, 'z': -0.040454, 'P1': 0.323678, 'P2': -0.091744, 'P3': -0.040454, 'P4': 0.075555, 'P5': 0.015928, 'P6': 0.072739, 'P7': 0.005061, 'P8': 0.015545, 'P9': 0.044210, 'P10': 0.004240 };
g_spherePositions['ICD.p23.f100.446009'] = { 'name': 'ICD.p23.f100.446009', 'color': 0, 'x': 0.599875, 'y': 0.045029, 'z': -0.254641, 'P1': 0.599875, 'P2': 0.045029, 'P3': -0.254641, 'P4': -0.100723, 'P5': 0.056148, 'P6': 0.048705, 'P7': 0.037376, 'P8': 0.061544, 'P9': -0.066593, 'P10': -0.006044 };
g_spherePositions['H.p16.dc35.445984'] = { 'name': 'H.p16.dc35.445984', 'color': 0, 'x': 0.005005, 'y': 0.105519, 'z': 0.007466, 'P1': 0.005005, 'P2': 0.105519, 'P3': 0.007466, 'P4': -0.026717, 'P5': 0.068205, 'P6': -0.020616, 'P7': -0.011459, 'P8': -0.043499, 'P9': -0.012493, 'P10': -0.045671 };
g_spherePositions['H.p18.dc36.446000'] = { 'name': 'H.p18.dc36.446000', 'color': 0, 'x': -0.255636, 'y': 0.087182, 'z': -0.011511, 'P1': -0.255636, 'P2': 0.087182, 'P3': -0.011511, 'P4': 0.034257, 'P5': 0.002583, 'P6': -0.000813, 'P7': 0.027703, 'P8': -0.006744, 'P9': -0.025688, 'P10': 0.003587 };
g_spherePositions['CCD.p17.dc34.446047'] = { 'name': 'CCD.p17.dc34.446047', 'color': 0, 'x': -0.434384, 'y': 0.052651, 'z': -0.098464, 'P1': -0.434384, 'P2': 0.052651, 'P3': -0.098464, 'P4': 0.048066, 'P5': -0.080079, 'P6': -0.048667, 'P7': -0.050551, 'P8': 0.072565, 'P9': -0.002756, 'P10': 0.015015 };
g_spherePositions['ICD.p18.dc33.446023'] = { 'name': 'ICD.p18.dc33.446023', 'color': 0, 'x': -0.337113, 'y': -0.175146, 'z': -0.043393, 'P1': -0.337113, 'P2': -0.175146, 'P3': -0.043393, 'P4': 0.025432, 'P5': 0.203130, 'P6': -0.007959, 'P7': 0.067614, 'P8': -0.078630, 'P9': 0.076334, 'P10': 0.004743 };
g_spherePositions['UC.p20.f43.446010'] = { 'name': 'UC.p20.f43.446010', 'color': 0, 'x': -0.017427, 'y': -0.048238, 'z': 0.111558, 'P1': -0.017427, 'P2': -0.048238, 'P3': 0.111558, 'P4': -0.058420, 'P5': 0.021245, 'P6': 0.019107, 'P7': 0.034341, 'P8': 0.027398, 'P9': -0.014839, 'P10': -0.030660 };
g_spherePositions['UC.p20.f40.445982'] = { 'name': 'UC.p20.f40.445982', 'color': 0, 'x': -0.132506, 'y': 0.068301, 'z': 0.079068, 'P1': -0.132506, 'P2': 0.068301, 'P3': 0.079068, 'P4': -0.040888, 'P5': -0.025599, 'P6': 0.037283, 'P7': 0.076075, 'P8': 0.003597, 'P9': 0.015370, 'P10': -0.038131 };
g_spherePositions['ICD.p22.f116.446042'] = { 'name': 'ICD.p22.f116.446042', 'color': 0, 'x': 0.288613, 'y': -0.009101, 'z': -0.067214, 'P1': 0.288613, 'P2': -0.009101, 'P3': -0.067214, 'P4': 0.062570, 'P5': -0.037602, 'P6': 0.040362, 'P7': 0.036188, 'P8': -0.042278, 'P9': 0.081759, 'P10': 0.062286 };
g_spherePositions['ICD.p22.f39.446093'] = { 'name': 'ICD.p22.f39.446093', 'color': 0, 'x': 0.294348, 'y': -0.135189, 'z': 0.033433, 'P1': 0.294348, 'P2': -0.135189, 'P3': 0.033433, 'P4': 0.070628, 'P5': -0.046931, 'P6': 0.128875, 'P7': -0.116044, 'P8': 0.061097, 'P9': 0.037283, 'P10': -0.117315 };
g_spherePositions['H.p13.f25.446075'] = { 'name': 'H.p13.f25.446075', 'color': 0, 'x': -0.073252, 'y': 0.080525, 'z': 0.080761, 'P1': -0.073252, 'P2': 0.080525, 'P3': 0.080761, 'P4': -0.016261, 'P5': -0.012073, 'P6': 0.035197, 'P7': 0.054393, 'P8': -0.017391, 'P9': -0.012148, 'P10': -0.007119 };
g_spherePositions['H.p14.f27.446096'] = { 'name': 'H.p14.f27.446096', 'color': 0, 'x': 0.101378, 'y': 0.024067, 'z': 0.082046, 'P1': 0.101378, 'P2': 0.024067, 'P3': 0.082046, 'P4': -0.021482, 'P5': -0.077881, 'P6': -0.009158, 'P7': -0.024749, 'P8': -0.016035, 'P9': 0.016302, 'P10': 0.002567 };
g_spherePositions['CCD.p14.f28.446040'] = { 'name': 'CCD.p14.f28.446040', 'color': 0, 'x': 0.118330, 'y': 0.036382, 'z': 0.048199, 'P1': 0.118330, 'P2': 0.036382, 'P3': 0.048199, 'P4': -0.003884, 'P5': -0.056618, 'P6': -0.059753, 'P7': -0.008710, 'P8': -0.044985, 'P9': -0.003650, 'P10': 0.010782 };
g_spherePositions['H.p11.f26.446031'] = { 'name': 'H.p11.f26.446031', 'color': 0, 'x': 0.283845, 'y': 0.111146, 'z': -0.036481, 'P1': 0.283845, 'P2': 0.111146, 'P3': -0.036481, 'P4': -0.180206, 'P5': -0.039440, 'P6': -0.108208, 'P7': -0.010746, 'P8': -0.032269, 'P9': -0.029900, 'P10': 0.012494 };
g_spherePositions['H.p1.f1.446017'] = { 'name': 'H.p1.f1.446017', 'color': 0, 'x': -0.042016, 'y': 0.032137, 'z': 0.080181, 'P1': -0.042016, 'P2': 0.032137, 'P3': 0.080181, 'P4': -0.061646, 'P5': -0.090872, 'P6': 0.060859, 'P7': -0.011070, 'P8': -0.001477, 'P9': 0.064634, 'P10': 0.039697 };
g_spherePositions['H.p6.f11.446043'] = { 'name': 'H.p6.f11.446043', 'color': 0, 'x': 0.256916, 'y': -0.000919, 'z': -0.015715, 'P1': 0.256916, 'P2': -0.000919, 'P3': -0.015715, 'P4': 0.056777, 'P5': -0.052127, 'P6': 0.004744, 'P7': -0.003141, 'P8': -0.039849, 'P9': 0.024404, 'P10': 0.066517 };
g_spherePositions['H.p1.f8.446044'] = { 'name': 'H.p1.f8.446044', 'color': 0, 'x': -0.130690, 'y': 0.108991, 'z': 0.021910, 'P1': -0.130690, 'P2': 0.108991, 'P3': 0.021910, 'P4': -0.021530, 'P5': 0.007180, 'P6': 0.044102, 'P7': 0.001794, 'P8': -0.028248, 'P9': 0.006128, 'P10': 0.012567 };
g_spherePositions['H.p6.f12.446024'] = { 'name': 'H.p6.f12.446024', 'color': 0, 'x': 0.378347, 'y': -0.022136, 'z': -0.009164, 'P1': 0.378347, 'P2': -0.022136, 'P3': -0.009164, 'P4': 0.141336, 'P5': -0.048390, 'P6': 0.011435, 'P7': 0.076501, 'P8': -0.004527, 'P9': -0.011058, 'P10': 0.021641 };
g_spherePositions['UC.p44.f54.446080'] = { 'name': 'UC.p44.f54.446080', 'color': 0, 'x': 0.272304, 'y': 0.026161, 'z': 0.011641, 'P1': 0.272304, 'P2': 0.026161, 'P3': 0.011641, 'P4': 0.043277, 'P5': -0.003857, 'P6': -0.047854, 'P7': -0.035008, 'P8': -0.019045, 'P9': -0.030889, 'P10': 0.001481 };
g_spherePositions['H.p44.f86.446048'] = { 'name': 'H.p44.f86.446048', 'color': 0, 'x': 0.314790, 'y': -0.002044, 'z': -0.059759, 'P1': 0.314790, 'P2': -0.002044, 'P3': -0.059759, 'P4': 0.103254, 'P5': -0.031767, 'P6': 0.022401, 'P7': 0.018031, 'P8': -0.023862, 'P9': 0.021166, 'P10': 0.055575 };
g_spherePositions['H.p35.f78.446089'] = { 'name': 'H.p35.f78.446089', 'color': 0, 'x': 0.056399, 'y': 0.082303, 'z': 0.056022, 'P1': 0.056399, 'P2': 0.082303, 'P3': 0.056022, 'P4': -0.110125, 'P5': -0.017388, 'P6': -0.017233, 'P7': -0.024441, 'P8': -0.028073, 'P9': -0.042890, 'P10': -0.008776 };
g_spherePositions['CCD.p35.f72.446033'] = { 'name': 'CCD.p35.f72.446033', 'color': 0, 'x': 0.300380, 'y': 0.116739, 'z': -0.082966, 'P1': 0.300380, 'P2': 0.116739, 'P3': -0.082966, 'P4': -0.225825, 'P5': -0.012510, 'P6': -0.082274, 'P7': -0.047100, 'P8': -0.026613, 'P9': 0.054559, 'P10': 0.036414 };
g_spherePositions['UC.p43.f51.445986'] = { 'name': 'UC.p43.f51.445986', 'color': 0, 'x': -0.225519, 'y': 0.105849, 'z': -0.019093, 'P1': -0.225519, 'P2': 0.105849, 'P3': -0.019093, 'P4': -0.032113, 'P5': -0.001398, 'P6': 0.014684, 'P7': 0.030540, 'P8': -0.016199, 'P9': -0.012914, 'P10': -0.021062 };
g_spherePositions['H.p43.f42.446087'] = { 'name': 'H.p43.f42.446087', 'color': 0, 'x': -0.159005, 'y': 0.103472, 'z': 0.013524, 'P1': -0.159005, 'P2': 0.103472, 'P3': 0.013524, 'P4': 0.002245, 'P5': 0.017866, 'P6': 0.037437, 'P7': 0.018109, 'P8': -0.029264, 'P9': -0.005747, 'P10': -0.006964 };
g_spherePositions['CCD.p11.f19.445990'] = { 'name': 'CCD.p11.f19.445990', 'color': 0, 'x': 0.023102, 'y': 0.063711, 'z': 0.076330, 'P1': 0.023102, 'P2': 0.063711, 'P3': 0.076330, 'P4': 0.011719, 'P5': -0.034869, 'P6': -0.029031, 'P7': -0.046906, 'P8': -0.040329, 'P9': -0.050732, 'P10': 0.003704 };
g_spherePositions['CCD.p9.f20.446011'] = { 'name': 'CCD.p9.f20.446011', 'color': 0, 'x': -0.057425, 'y': 0.016541, 'z': 0.075268, 'P1': -0.057425, 'P2': 0.016541, 'P3': 0.075268, 'P4': -0.045091, 'P5': -0.050832, 'P6': 0.061728, 'P7': -0.011870, 'P8': -0.000040, 'P9': -0.039039, 'P10': 0.036635 };
g_spherePositions['ICD.p10.f18.446085'] = { 'name': 'ICD.p10.f18.446085', 'color': 0, 'x': 0.418786, 'y': 0.091481, 'z': -0.157449, 'P1': 0.418786, 'P2': 0.091481, 'P3': -0.157449, 'P4': -0.062323, 'P5': 0.081301, 'P6': -0.010655, 'P7': -0.007585, 'P8': 0.032207, 'P9': 0.030163, 'P10': 0.067669 };
g_spherePositions['CCD.p9.f17.446027'] = { 'name': 'CCD.p9.f17.446027', 'color': 0, 'x': -0.103559, 'y': 0.034599, 'z': 0.051408, 'P1': -0.103559, 'P2': 0.034599, 'P3': 0.051408, 'P4': -0.052325, 'P5': -0.074337, 'P6': 0.117427, 'P7': 0.006997, 'P8': 0.034121, 'P9': 0.014084, 'P10': 0.029290 };
g_spherePositions['CCD.p14.b28.446036'] = { 'name': 'CCD.p14.b28.446036', 'color': 0, 'x': -0.274279, 'y': 0.100495, 'z': -0.041069, 'P1': -0.274279, 'P2': 0.100495, 'P3': -0.041069, 'P4': 0.066565, 'P5': 0.001700, 'P6': -0.013827, 'P7': 0.019961, 'P8': -0.009048, 'P9': -0.025447, 'P10': 0.012756 };
g_spherePositions['H.p14.b27.446004'] = { 'name': 'H.p14.b27.446004', 'color': 0, 'x': -0.305585, 'y': 0.114755, 'z': -0.053351, 'P1': -0.305585, 'P2': 0.114755, 'P3': -0.053351, 'P4': 0.085848, 'P5': 0.001044, 'P6': -0.032626, 'P7': 0.018174, 'P8': -0.011839, 'P9': -0.016101, 'P10': 0.010265 };
g_spherePositions['H.p13.b25.446019'] = { 'name': 'H.p13.b25.446019', 'color': 0, 'x': -0.282706, 'y': 0.101043, 'z': -0.053198, 'P1': -0.282706, 'P2': 0.101043, 'P3': -0.053198, 'P4': 0.105537, 'P5': 0.052799, 'P6': -0.011554, 'P7': 0.027295, 'P8': -0.007080, 'P9': -0.023339, 'P10': 0.000129 };
g_spherePositions['H.p11.b26.446032'] = { 'name': 'H.p11.b26.446032', 'color': 0, 'x': 0.143566, 'y': 0.078092, 'z': 0.025447, 'P1': 0.143566, 'P2': 0.078092, 'P3': 0.025447, 'P4': 0.093802, 'P5': 0.011935, 'P6': -0.122159, 'P7': -0.055979, 'P8': -0.085817, 'P9': -0.011053, 'P10': 0.000234 };
g_spherePositions['CCD.p17.i32.446060'] = { 'name': 'CCD.p17.i32.446060', 'color': 0, 'x': -0.420710, 'y': 0.040762, 'z': -0.111826, 'P1': -0.420710, 'P2': 0.040762, 'P3': -0.111826, 'P4': 0.059124, 'P5': -0.036172, 'P6': -0.028279, 'P7': -0.067015, 'P8': 0.066185, 'P9': -0.000284, 'P10': 0.017484 };
g_spherePositions['CCD.p17.i34.446068'] = { 'name': 'CCD.p17.i34.446068', 'color': 0, 'x': -0.398700, 'y': 0.062562, 'z': -0.083623, 'P1': -0.398700, 'P2': 0.062562, 'P3': -0.083623, 'P4': 0.052517, 'P5': -0.062635, 'P6': -0.047299, 'P7': -0.030413, 'P8': 0.049779, 'P9': -0.010512, 'P10': 0.020287 };
g_spherePositions['ICD.p18.i33.446045'] = { 'name': 'ICD.p18.i33.446045', 'color': 0, 'x': -0.273166, 'y': -0.656806, 'z': -0.140166, 'P1': -0.273166, 'P2': -0.656806, 'P3': -0.140166, 'P4': -0.040401, 'P5': -0.008133, 'P6': -0.010528, 'P7': -0.019476, 'P8': -0.150467, 'P9': -0.070714, 'P10': 0.085603 };
g_spherePositions['H.p18.f36.445998'] = { 'name': 'H.p18.f36.445998', 'color': 0, 'x': -0.040487, 'y': -0.211506, 'z': 0.159037, 'P1': -0.040487, 'P2': -0.211506, 'P3': 0.159037, 'P4': -0.112497, 'P5': 0.005988, 'P6': 0.013982, 'P7': 0.097314, 'P8': 0.045518, 'P9': -0.009619, 'P10': 0.060405 };
g_spherePositions['CCD.p17.f34.445993'] = { 'name': 'CCD.p17.f34.445993', 'color': 0, 'x': -0.087329, 'y': 0.063932, 'z': 0.051611, 'P1': -0.087329, 'P2': 0.063932, 'P3': 0.051611, 'P4': -0.010553, 'P5': -0.007739, 'P6': 0.044330, 'P7': -0.016226, 'P8': -0.013610, 'P9': -0.036761, 'P10': 0.025504 };
g_spherePositions['H.p16.f35.446041'] = { 'name': 'H.p16.f35.446041', 'color': 0, 'x': 0.058675, 'y': 0.070058, 'z': 0.066740, 'P1': 0.058675, 'P2': 0.070058, 'P3': 0.066740, 'P4': -0.001691, 'P5': -0.021340, 'P6': -0.033290, 'P7': 0.021654, 'P8': -0.032226, 'P9': -0.004976, 'P10': -0.029381 };
g_spherePositions['ICD.p18.f33.446006'] = { 'name': 'ICD.p18.f33.446006', 'color': 0, 'x': 0.433129, 'y': -0.123402, 'z': -0.064556, 'P1': 0.433129, 'P2': -0.123402, 'P3': -0.064556, 'P4': 0.162680, 'P5': 0.037653, 'P6': 0.053271, 'P7': 0.035238, 'P8': 0.034790, 'P9': -0.030375, 'P10': -0.085155 };
g_spherePositions['H.p25.f58.446081'] = { 'name': 'H.p25.f58.446081', 'color': 0, 'x': -0.129131, 'y': 0.167011, 'z': -0.088842, 'P1': -0.129131, 'P2': 0.167011, 'P3': -0.088842, 'P4': -0.218790, 'P5': 0.055340, 'P6': 0.036226, 'P7': 0.045691, 'P8': -0.017820, 'P9': -0.017096, 'P10': -0.033977 };
g_spherePositions['H.p27.f103.446077'] = { 'name': 'H.p27.f103.446077', 'color': 0, 'x': -0.013525, 'y': 0.060164, 'z': 0.084370, 'P1': -0.013525, 'P2': 0.060164, 'P3': 0.084370, 'P4': -0.010155, 'P5': -0.019921, 'P6': 0.009789, 'P7': 0.017639, 'P8': -0.025693, 'P9': -0.006404, 'P10': -0.014207 };
g_spherePositions['UC.p25.f115.446095'] = { 'name': 'UC.p25.f115.446095', 'color': 0, 'x': -0.028283, 'y': 0.033938, 'z': 0.116142, 'P1': -0.028283, 'P2': 0.033938, 'P3': 0.116142, 'P4': -0.057548, 'P5': -0.106141, 'P6': 0.026494, 'P7': 0.009763, 'P8': 0.008587, 'P9': 0.068542, 'P10': -0.035524 };
g_spherePositions['ICD.p27.f64.446073'] = { 'name': 'ICD.p27.f64.446073', 'color': 0, 'x': 0.010559, 'y': -0.069444, 'z': 0.095277, 'P1': 0.010559, 'P2': -0.069444, 'P3': 0.095277, 'P4': -0.000040, 'P5': 0.021946, 'P6': 0.221879, 'P7': -0.094258, 'P8': 0.035294, 'P9': -0.145754, 'P10': 0.094124 };
g_spherePositions['H.p39.f105.446035'] = { 'name': 'H.p39.f105.446035', 'color': 0, 'x': 0.195983, 'y': -0.180240, 'z': 0.108287, 'P1': 0.195983, 'P2': -0.180240, 'P3': 0.108287, 'P4': 0.037438, 'P5': 0.067886, 'P6': -0.134283, 'P7': 0.033462, 'P8': 0.068176, 'P9': -0.069205, 'P10': -0.007627 };
g_spherePositions['CCD.p31.f41.446029'] = { 'name': 'CCD.p31.f41.446029', 'color': 0, 'x': -0.307549, 'y': 0.049329, 'z': -0.001775, 'P1': -0.307549, 'P2': 0.049329, 'P3': -0.001775, 'P4': -0.008108, 'P5': -0.104362, 'P6': -0.035936, 'P7': 0.051778, 'P8': 0.029004, 'P9': 0.005615, 'P10': -0.025340 };
g_spherePositions['H.p31.f61.446007'] = { 'name': 'H.p31.f61.446007', 'color': 0, 'x': 0.338472, 'y': -0.000412, 'z': -0.049484, 'P1': 0.338472, 'P2': -0.000412, 'P3': -0.049484, 'P4': 0.115361, 'P5': -0.037313, 'P6': -0.003130, 'P7': 0.063114, 'P8': -0.016697, 'P9': -0.048137, 'P10': 0.022299 };
g_spherePositions['UC.p39.f104.446074'] = { 'name': 'UC.p39.f104.446074', 'color': 0, 'x': 0.200676, 'y': 0.049679, 'z': -0.001118, 'P1': 0.200676, 'P2': 0.049679, 'P3': -0.001118, 'P4': 0.079396, 'P5': 0.020916, 'P6': -0.065789, 'P7': -0.008840, 'P8': -0.046644, 'P9': 0.062164, 'P10': 0.001805 };
g_spherePositions['H.p33.f93.445992'] = { 'name': 'H.p33.f93.445992', 'color': 0, 'x': -0.306517, 'y': -0.084895, 'z': 0.087523, 'P1': -0.306517, 'P2': -0.084895, 'P3': 0.087523, 'P4': -0.095919, 'P5': 0.056073, 'P6': -0.038999, 'P7': 0.144725, 'P8': 0.023629, 'P9': -0.001203, 'P10': -0.115707 };
g_spherePositions['ICD.p33.f83.446086'] = { 'name': 'ICD.p33.f83.446086', 'color': 0, 'x': -0.032959, 'y': -0.162254, 'z': 0.134997, 'P1': -0.032959, 'P2': -0.162254, 'P3': 0.134997, 'P4': -0.061405, 'P5': 0.095923, 'P6': 0.045147, 'P7': 0.037343, 'P8': 0.057899, 'P9': -0.021766, 'P10': 0.104849 };
g_spherePositions['H.p41.f87.446038'] = { 'name': 'H.p41.f87.446038', 'color': 0, 'x': 0.154712, 'y': 0.038941, 'z': 0.047191, 'P1': 0.154712, 'P2': 0.038941, 'P3': 0.047191, 'P4': 0.011539, 'P5': -0.020542, 'P6': -0.080644, 'P7': -0.009789, 'P8': -0.041149, 'P9': 0.039971, 'P10': -0.040618 };
g_spherePositions['UC.p41.f102.446005'] = { 'name': 'UC.p41.f102.446005', 'color': 0, 'x': -0.231424, 'y': 0.090101, 'z': 0.014161, 'P1': -0.231424, 'P2': 0.090101, 'P3': 0.014161, 'P4': 0.003784, 'P5': -0.033291, 'P6': 0.002044, 'P7': 0.060149, 'P8': 0.000575, 'P9': -0.015658, 'P10': -0.010656 };
g_spherePositions['H.p34.f75.446051'] = { 'name': 'H.p34.f75.446051', 'color': 0, 'x': 0.331995, 'y': -0.010713, 'z': -0.062931, 'P1': 0.331995, 'P2': -0.010713, 'P3': -0.062931, 'P4': 0.084260, 'P5': -0.061997, 'P6': 0.031226, 'P7': 0.053232, 'P8': -0.015851, 'P9': 0.028067, 'P10': 0.032646 };
g_spherePositions['CCD.p34.f67.446052'] = { 'name': 'CCD.p34.f67.446052', 'color': 0, 'x': -0.017566, 'y': -0.062690, 'z': 0.139268, 'P1': -0.017566, 'P2': -0.062690, 'P3': 0.139268, 'P4': -0.053745, 'P5': 0.021847, 'P6': -0.015308, 'P7': 0.041323, 'P8': 0.027670, 'P9': -0.005587, 'P10': 0.006345 };
g_spherePositions['UC.p42.f76.446066'] = { 'name': 'UC.p42.f76.446066', 'color': 0, 'x': -0.070482, 'y': 0.101745, 'z': 0.057103, 'P1': -0.070482, 'P2': 0.101745, 'P3': 0.057103, 'P4': 0.020608, 'P5': 0.018247, 'P6': 0.011014, 'P7': 0.037462, 'P8': -0.034477, 'P9': 0.006639, 'P10': -0.025629 };
g_spherePositions['H.p42.f79.446022'] = { 'name': 'H.p42.f79.446022', 'color': 0, 'x': 0.115910, 'y': 0.022586, 'z': 0.069057, 'P1': 0.115910, 'P2': 0.022586, 'P3': 0.069057, 'P4': -0.014404, 'P5': -0.047329, 'P6': -0.031912, 'P7': -0.030371, 'P8': -0.025995, 'P9': 0.007733, 'P10': -0.020498 };
g_spherePositions['CCD.p11.dc19.446067'] = { 'name': 'CCD.p11.dc19.446067', 'color': 0, 'x': -0.086258, 'y': 0.082294, 'z': 0.034700, 'P1': -0.086258, 'P2': 0.082294, 'P3': 0.034700, 'P4': 0.021546, 'P5': 0.041308, 'P6': 0.021227, 'P7': -0.014219, 'P8': -0.042995, 'P9': -0.029687, 'P10': -0.015519 };
g_spherePositions['ICD.p10.dc18.445983'] = { 'name': 'ICD.p10.dc18.445983', 'color': 0, 'x': -0.141504, 'y': 0.165693, 'z': -0.146519, 'P1': -0.141504, 'P2': 0.165693, 'P3': -0.146519, 'P4': -0.198145, 'P5': 0.118176, 'P6': 0.068405, 'P7': -0.039987, 'P8': -0.052973, 'P9': -0.011005, 'P10': 0.010590 };
g_spherePositions['UC.p37.f52.446039'] = { 'name': 'UC.p37.f52.446039', 'color': 0, 'x': 0.226753, 'y': 0.030396, 'z': 0.037151, 'P1': 0.226753, 'P2': 0.030396, 'P3': 0.037151, 'P4': 0.070308, 'P5': -0.011401, 'P6': -0.070752, 'P7': -0.012278, 'P8': -0.027350, 'P9': 0.032948, 'P10': -0.003127 };
g_spherePositions['ICD.p29.f110.446003'] = { 'name': 'ICD.p29.f110.446003', 'color': 0, 'x': -0.327401, 'y': -0.691353, 'z': -0.053944, 'P1': -0.327401, 'P2': -0.691353, 'P3': -0.053944, 'P4': -0.117840, 'P5': -0.032943, 'P6': -0.082794, 'P7': -0.013118, 'P8': -0.051436, 'P9': 0.067474, 'P10': 0.029448 };
g_spherePositions['H.p37.f113.446084'] = { 'name': 'H.p37.f113.446084', 'color': 0, 'x': -0.120413, 'y': 0.110178, 'z': -0.018573, 'P1': -0.120413, 'P2': 0.110178, 'P3': -0.018573, 'P4': -0.149628, 'P5': 0.038387, 'P6': 0.074165, 'P7': -0.023456, 'P8': 0.003105, 'P9': -0.035592, 'P10': 0.059809 };
g_spherePositions['H.p29.f48.446015'] = { 'name': 'H.p29.f48.446015', 'color': 0, 'x': 0.064536, 'y': -0.009359, 'z': 0.105424, 'P1': 0.064536, 'P2': -0.009359, 'P3': 0.105424, 'P4': -0.087020, 'P5': -0.094190, 'P6': 0.052262, 'P7': -0.000676, 'P8': 0.019214, 'P9': 0.012679, 'P10': -0.047930 };
g_spherePositions['ICD.p15.f29.445999'] = { 'name': 'ICD.p15.f29.445999', 'color': 0, 'x': 0.150516, 'y': -0.236243, 'z': 0.062951, 'P1': 0.150516, 'P2': -0.236243, 'P3': 0.062951, 'P4': -0.038179, 'P5': 0.113550, 'P6': -0.053147, 'P7': 0.007188, 'P8': 0.095288, 'P9': 0.087036, 'P10': 0.036498 };
g_spherePositions['ICD.p16.f31.446054'] = { 'name': 'ICD.p16.f31.446054', 'color': 0, 'x': 0.272218, 'y': -0.080894, 'z': 0.032005, 'P1': 0.272218, 'P2': -0.080894, 'P3': 0.032005, 'P4': 0.100517, 'P5': 0.053423, 'P6': -0.026494, 'P7': -0.127345, 'P8': 0.043353, 'P9': -0.022454, 'P10': -0.002999 };
g_spherePositions['ICD.p15.f30.446076'] = { 'name': 'ICD.p15.f30.446076', 'color': 0, 'x': 0.303439, 'y': 0.055559, 'z': -0.063220, 'P1': 0.303439, 'P2': 0.055559, 'P3': -0.063220, 'P4': -0.136198, 'P5': 0.107771, 'P6': -0.016748, 'P7': -0.126991, 'P8': 0.028880, 'P9': 0.050271, 'P10': 0.026755 };
g_spherePositions['CCD.p17.f32.446063'] = { 'name': 'CCD.p17.f32.446063', 'color': 0, 'x': 0.113358, 'y': 0.010353, 'z': 0.088812, 'P1': 0.113358, 'P2': 0.010353, 'P3': 0.088812, 'P4': 0.017393, 'P5': -0.064489, 'P6': 0.015928, 'P7': -0.073686, 'P8': -0.028961, 'P9': 0.048525, 'P10': 0.049831 };
g_spherePositions['H.p40.f81.446055'] = { 'name': 'H.p40.f81.446055', 'color': 0, 'x': 0.422917, 'y': -0.015523, 'z': -0.060913, 'P1': 0.422917, 'P2': -0.015523, 'P3': -0.060913, 'P4': 0.173169, 'P5': -0.056339, 'P6': -0.023023, 'P7': 0.078494, 'P8': -0.001757, 'P9': -0.062500, 'P10': -0.037202 };
g_spherePositions['uiCD.p32.f91.446020'] = { 'name': 'uiCD.p32.f91.446020', 'color': 0, 'x': 0.359617, 'y': -0.055220, 'z': -0.063875, 'P1': 0.359617, 'P2': -0.055220, 'P3': -0.063875, 'P4': 0.114222, 'P5': -0.028952, 'P6': 0.082829, 'P7': 0.027458, 'P8': 0.014547, 'P9': 0.005145, 'P10': 0.014231 };
g_spherePositions['UC.p40.f73.446021'] = { 'name': 'UC.p40.f73.446021', 'color': 0, 'x': 0.408312, 'y': -0.021635, 'z': -0.057986, 'P1': 0.408312, 'P2': -0.021635, 'P3': -0.057986, 'P4': 0.157180, 'P5': -0.023376, 'P6': 0.039497, 'P7': 0.044099, 'P8': 0.004721, 'P9': -0.012190, 'P10': 0.010418 };
g_spherePositions['H.p32.f98.446030'] = { 'name': 'H.p32.f98.446030', 'color': 0, 'x': 0.094467, 'y': 0.017883, 'z': 0.106819, 'P1': 0.094467, 'P2': 0.017883, 'P3': 0.106819, 'P4': -0.036791, 'P5': -0.067637, 'P6': -0.006717, 'P7': -0.024060, 'P8': 0.006801, 'P9': 0.023728, 'P10': -0.015582 };
g_spherePositions['H.p12.f22.446097'] = { 'name': 'H.p12.f22.446097', 'color': 0, 'x': -0.247660, 'y': 0.036489, 'z': 0.016449, 'P1': -0.247660, 'P2': 0.036489, 'P3': 0.016449, 'P4': -0.041241, 'P5': -0.104099, 'P6': 0.020430, 'P7': 0.041971, 'P8': 0.031155, 'P9': 0.036944, 'P10': 0.015731 };
g_spherePositions['ICD.p10.f21.445991'] = { 'name': 'ICD.p10.f21.445991', 'color': 0, 'x': -0.246426, 'y': 0.046340, 'z': -0.002331, 'P1': -0.246426, 'P2': 0.046340, 'P3': -0.002331, 'P4': -0.004892, 'P5': -0.035263, 'P6': 0.045270, 'P7': 0.049608, 'P8': 0.023743, 'P9': 0.054136, 'P10': 0.013141 };
g_spherePositions['CCD.p12.f23.446016'] = { 'name': 'CCD.p12.f23.446016', 'color': 0, 'x': 0.160524, 'y': 0.019310, 'z': 0.031082, 'P1': 0.160524, 'P2': 0.019310, 'P3': 0.031082, 'P4': 0.049586, 'P5': -0.018643, 'P6': -0.061038, 'P7': -0.038610, 'P8': -0.058239, 'P9': -0.026177, 'P10': 0.023051 };
g_spherePositions['CCD.p13.f24.445989'] = { 'name': 'CCD.p13.f24.445989', 'color': 0, 'x': -0.224174, 'y': 0.081447, 'z': 0.018400, 'P1': -0.224174, 'P2': 0.081447, 'P3': 0.018400, 'P4': 0.028816, 'P5': -0.053426, 'P6': 0.018277, 'P7': 0.082666, 'P8': -0.009674, 'P9': -0.008913, 'P10': 0.028067 };
g_spherePositions['CCD.p12.dc23.446065'] = { 'name': 'CCD.p12.dc23.446065', 'color': 0, 'x': -0.451929, 'y': 0.068367, 'z': -0.133141, 'P1': -0.451929, 'P2': 0.068367, 'P3': -0.133141, 'P4': 0.069532, 'P5': -0.063976, 'P6': -0.063917, 'P7': -0.019092, 'P8': 0.086351, 'P9': 0.032183, 'P10': 0.012456 };
g_spherePositions['CCD.p13.dc24.446026'] = { 'name': 'CCD.p13.dc24.446026', 'color': 0, 'x': -0.287577, 'y': 0.072330, 'z': -0.038734, 'P1': -0.287577, 'P2': 0.072330, 'P3': -0.038734, 'P4': 0.077560, 'P5': 0.043631, 'P6': -0.011615, 'P7': 0.027654, 'P8': -0.017834, 'P9': -0.041917, 'P10': -0.002251 };
g_spherePositions['H.p12.dc22.446083'] = { 'name': 'H.p12.dc22.446083', 'color': 0, 'x': -0.488608, 'y': 0.044612, 'z': -0.128430, 'P1': -0.488608, 'P2': 0.044612, 'P3': -0.128430, 'P4': 0.036082, 'P5': -0.146535, 'P6': -0.083748, 'P7': -0.030082, 'P8': 0.121777, 'P9': 0.006799, 'P10': -0.018863 };
g_spherePositions['UC.p38.f38.446094'] = { 'name': 'UC.p38.f38.446094', 'color': 0, 'x': 0.267629, 'y': -0.091523, 'z': 0.040485, 'P1': 0.267629, 'P2': -0.091523, 'P3': 0.040485, 'P4': 0.046240, 'P5': 0.001337, 'P6': -0.018546, 'P7': 0.030875, 'P8': 0.024279, 'P9': -0.006959, 'P10': -0.002920 };
g_spherePositions['ICCD.p30.f57.446056'] = { 'name': 'ICCD.p30.f57.446056', 'color': 0, 'x': -0.254120, 'y': 0.032587, 'z': -0.042535, 'P1': -0.254120, 'P2': 0.032587, 'P3': -0.042535, 'P4': 0.007681, 'P5': 0.074435, 'P6': 0.082017, 'P7': 0.041952, 'P8': -0.013290, 'P9': 0.117969, 'P10': -0.043678 };
g_spherePositions['H.p30.f85.445985'] = { 'name': 'H.p30.f85.445985', 'color': 0, 'x': 0.230356, 'y': -0.097364, 'z': 0.027917, 'P1': 0.230356, 'P2': -0.097364, 'P3': 0.027917, 'P4': 0.002256, 'P5': -0.050317, 'P6': 0.036925, 'P7': -0.101490, 'P8': 0.020505, 'P9': 0.024977, 'P10': -0.022009 };
g_spherePositions['H.p38.f63.445988'] = { 'name': 'H.p38.f63.445988', 'color': 0, 'x': 0.147617, 'y': 0.006975, 'z': 0.048031, 'P1': 0.147617, 'P2': 0.006975, 'P3': 0.048031, 'P4': -0.024116, 'P5': -0.084439, 'P6': 0.013117, 'P7': 0.000035, 'P8': -0.032686, 'P9': 0.071148, 'P10': 0.037904 };
g_spherePositions['UC.p46.f71.446071'] = { 'name': 'UC.p46.f71.446071', 'color': 0, 'x': 0.146509, 'y': 0.010414, 'z': 0.114596, 'P1': 0.146509, 'P2': 0.010414, 'P3': 0.114596, 'P4': -0.035783, 'P5': -0.037417, 'P6': -0.003320, 'P7': -0.051797, 'P8': 0.016983, 'P9': 0.037275, 'P10': -0.055364 };
g_spherePositions['UC.p45.f66.446001'] = { 'name': 'UC.p45.f66.446001', 'color': 0, 'x': 0.015623, 'y': 0.053531, 'z': 0.086587, 'P1': 0.015623, 'P2': 0.053531, 'P3': 0.086587, 'P4': -0.000897, 'P5': -0.053214, 'P6': -0.004166, 'P7': -0.012645, 'P8': -0.021324, 'P9': -0.057415, 'P10': -0.012070 };
g_spherePositions['H.p46.f69.446078'] = { 'name': 'H.p46.f69.446078', 'color': 0, 'x': 0.686314, 'y': 0.044831, 'z': -0.336908, 'P1': 0.686314, 'P2': 0.044831, 'P3': -0.336908, 'P4': -0.207430, 'P5': -0.046818, 'P6': 0.029179, 'P7': 0.102808, 'P8': 0.054111, 'P9': -0.016173, 'P10': -0.060219 };
g_spherePositions['H.p45.f80.446018'] = { 'name': 'H.p45.f80.446018', 'color': 0, 'x': 0.084202, 'y': 0.035280, 'z': 0.073653, 'P1': 0.084202, 'P2': 0.035280, 'P3': 0.073653, 'P4': -0.015927, 'P5': -0.067223, 'P6': -0.041718, 'P7': -0.037926, 'P8': -0.032728, 'P9': -0.031397, 'P10': -0.006744 };
g_spherePositions['ICD.p10.i21.446062'] = { 'name': 'ICD.p10.i21.446062', 'color': 0, 'x': -0.348653, 'y': 0.074996, 'z': -0.109253, 'P1': -0.348653, 'P2': 0.074996, 'P3': -0.109253, 'P4': 0.085495, 'P5': 0.004444, 'P6': 0.037462, 'P7': -0.002691, 'P8': 0.001181, 'P9': 0.121201, 'P10': 0.044500 };
g_spherePositions['H.p11.i26.446092'] = { 'name': 'H.p11.i26.446092', 'color': 0, 'x': -0.112815, 'y': -0.099300, 'z': 0.030847, 'P1': -0.112815, 'P2': -0.099300, 'P3': 0.030847, 'P4': -0.018775, 'P5': 0.052632, 'P6': -0.003985, 'P7': -0.037299, 'P8': 0.077524, 'P9': -0.113114, 'P10': 0.033116 };
g_spherePositions['CCD.p11.i19.446012'] = { 'name': 'CCD.p11.i19.446012', 'color': 0, 'x': 0.069181, 'y': 0.022969, 'z': 0.043220, 'P1': 0.069181, 'P2': 0.022969, 'P3': 0.043220, 'P4': 0.030075, 'P5': 0.041673, 'P6': -0.052348, 'P7': -0.059873, 'P8': -0.014765, 'P9': -0.030380, 'P10': -0.015850 };
g_spherePositions['ICD.p10.i18.445987'] = { 'name': 'ICD.p10.i18.445987', 'color': 0, 'x': -0.396645, 'y': 0.043672, 'z': -0.178219, 'P1': -0.396645, 'P2': 0.043672, 'P3': -0.178219, 'P4': -0.044620, 'P5': -0.001316, 'P6': -0.006837, 'P7': -0.083153, 'P8': 0.058971, 'P9': 0.000848, 'P10': -0.001006 };

var g_ellipsesDimensions = new Array();
var g_segments = 8, g_rings = 8, g_radius = 0.014099;
var g_xAxisLength = 1.174922;
var g_yAxisLength = 0.858365;
var g_zAxisLength = 0.495946;
var g_xMaximumValue = 0.686314;
var g_yMaximumValue = 0.167011;
var g_zMaximumValue = 0.159037;
var g_xMinimumValue = -0.488608;
var g_yMinimumValue = -0.691353;
var g_zMinimumValue = -0.336908;
var g_maximum = 0.691353;
var g_pc1Label = "PC1 (46 %)";
var g_pc2Label = "PC2 (14 %)";
var g_pc3Label = "PC3 (5 %)";
var g_number_of_custom_axes = 0;
var g_fractionExplained = [0.457469, 0.138819, 0.053418, 0.049108, 0.030591, 0.021126, 0.016370, 0.013655, 0.013172, 0.011758];
var g_fractionExplainedRounded = [46, 14, 5, 5, 3, 2, 2, 1, 1, 1];

var g_taxaPositions = new Array();


var g_vectorPositions = new Array();

var g_comparisonPositions = new Array();
var g_isSerialComparisonPlot = true;
document.getElementById("logo").style.display = 'none';
document.getElementById("logotable").style.display = 'none';

 </script>
</head>

<body>

<div id="plotToggle">
    <form>
      <div id="plottype">
        <input id="pcoa" type="radio" id="pcoa" name="plottype" checked="checked" /><label for="pcoa">PCoA</label>
        <input id="parallel" type="radio" id="parallel" name="plottype" /><label for="parallel">Parallel</label>
      </div>
    </form>
</div>
<div id="pcoaPlotWrapper" class="plotWrapper">
    <label id="pointCount" class="ontop">
    </label>

    <div id="finder" class="arrow-right">
    </div>

    <div id="labels" class="unselectable">
    </div>

    <div id="taxalabels" class="unselectable">
    </div>

    <div id="axislabels" class="axislabels">
    </div>

    <div id="main_plot">
    </div>
</div>

<div id="parallelPlotWrapper" class="plotWrapper">
</div>

<div class="separator" ondblclick="separatorDoubleClick()"></div>

<div id="menu">
    <div id="menutabs">
        <ul>
            <li><a href="#keytab">Key</a></li>
            <li><a href="#colorby">Colors</a></li>
            <li><a href="#showby">Visibility</a></li>
            <li><a href="#scalingby">Scaling</a></li>
            <li><a href="#labelby">Labels</a></li>
            <li><a href="#axes">Axes</a></li>
            <li><a href="#options">Options</a></li>
        </ul>
        <div id="keytab">
            <form name="keyFilter">
            <label>Filter  </label><input name="filterBox" id="searchBox" type="text" onkeyup="filterKey()"></input>
            </form>
            <div id="key">
            </div>
        </div>
        <div id="colorby">
            <br>
            <input type="checkbox" onchange="toggleContinuousAndDiscreteColors(this)" id="discreteorcontinuouscolors" name="discreteorcontinuouscolors">  Use gradient colors</input>
            <br><br>
            <select id="colorbycombo" onchange="colorByMenuChanged()" size="3">
            </select>
            <div class="list" id="colorbylist">
            </div>
        </div>
        <div id="showby" align="center">
            <table width="100%">
                <tr>
                    <td align="center">
                        <select id="showbycombo" onchange="showByMenuChanged()">
                        </select>
                    </td>
                </tr>
                <tr>
                    <td>
                        <div class="list" id="showbylist" style="height:100%;width:100%">
                        </div>
                    </td>
                </tr>
                <tr>
                    <td style="padding-left: 12px; padding-right:12px;">
                        <br>
                        <label for="sphereopacity" class="text">Global Sphere Opacity</label>
                        <label id="sphereopacity" class="slidervalue"></label>
                        <div id="sopacityslider" class="slider-range-max"></div>
                    </td>
                </tr>
            </table>
        </div>
        <div id="scalingby" align="center">
            <table width="100%">
                <tr>
                    <td align="center">
                        <select id="scalingbycombo" onchange="scalingByMenuChanged()">
                        </select>
                    </td>
                </tr>
                <tr>
                    <td>
                        <div class="list" id="scalingbylist" style="height:100%;width:100%">
                        </div>
                    </td>
                </tr>
                <tr>
                    <td style="padding-left: 12px; padding-right:12px;">
                        <br>
                        <label for="sphereradius" class="text">Global Sphere Scale</label>
                        <label id="sphereradius" class="slidervalue"></label>
                        <div id="sradiusslider" class="slider-range-max"></div>
                    </td>
                </tr>
            </table>
        </div>
        <div id="labelby">
        <div id="labelsTop">
            <form name="plotoptions">
            <input type="checkbox" onClick="toggleLabels()">Samples Label Visibility</input>
            </form>
            <br>
            <label for="labelopacity" class="text">Label Opacity</label>
            <label id="labelopacity" class="slidervalue"></label>
            <div id="lopacityslider" class="slider-range-max"></div>
            <div id="labelColorHolder clearfix">
            <table>
                <tr><td><div id="labelColor" class="colorbox"></div></td><td><label>Master Label Color</label></td></tr>
            </table></div>
        </div>
            <br>
            <select id="labelcombo" onchange="labelMenuChanged()">
            </select>
            <div class="list" id="labellist">
            </div>
        </div>
        <div id="axes">
            <div id="pcoaaxes">
                <div class="list" id="axeslist">
                </div>
            </div>
        </div>
        <div id="options">
            <table>
                <tr><td><div id="axeslabelscolor" class="colorbox" name="axeslabelscolor"></div></td><td title="Axes Labels Color">Axes Labels Color</td></tr>
                <tr><td><div id="axescolor" class="colorbox" name="axescolor"></div></td><td title="Axes Color Title">Axes Color</td></tr>
                <tr><td><div id="rendererbackgroundcolor" class="colorbox" name="rendererbackgroundcolor"></div></td><td title="Background Color Title">Background Color</td></tr>
            </table>
            <div id="pcoaviewoptions" class="">
                <form name="settingsoptionscolor">
                </form>
                <div id="pcoaoptions" class="">
                    <form name="settingsoptions">
                        <input type="checkbox" onchange="toggleScaleCoordinates(this)" id="scale_checkbox" name="scale_checkbox">Scale coords by percent explained</input>
                    </form>
                </div>
                <br><input id="reset" class="button" type="submit" value="Recenter Camera" style="" onClick="resetCamera()">
                <br><br>
                <hr class='section-break'>
                <br>Filename <small>(only letters, numbers, ., - and _)</small>:
                <br><input name="saveas_name" id="saveas_name" value="screenshot" type="text"/>
                <br><input id="saveas_legends" class="checkbox" type="checkbox" style=""> Create legend
                <input id="saveas" class="button" type="submit" value="Save as SVG" style="" onClick="saveSVG()"/>
                <br><br>For a PNG, simply press 'ctrl+p'.
                <div id="paralleloptions" class="">
                </div>
            </div>
            <br>
        </div>
    </div>  
</div>
</body>

</html>
